data/toroid-wedge-o3-s3.mesh

```diff
@@ -0,0 +1,236 @@
+MFEM mesh v1.0
```

Comment:
Rename to `toroid-wedge.mesh`, consider adding also `toroid-hex.mesh`.

Reply:
Done